source/agora/consensus/EnrollmentPool.d

```diff
@@ -41,19 +42,24 @@ public class EnrollmentPool
     /// SQLite DB instance
     private ManagedDatabase db;
 
+    /// Parameters for consensus-critical constants
+    private immutable(ConsensusParams) params;
```

Comment:
I think it's not needed in this class. I can't think of any consensus parameters that would affect how the pool behaves.

Reply:
The consensus-critical constants are protocol-level values, so it must affect all the objects in the network. But the pool does not use this value now. so it can be deleted. I will delete it.